Automattic/newspack-plugin
# feat: remove support wizard

### All Submissions:






### Changes proposed in this Pull Request:



Removes the Support wizard. This wizard allowed users to view and submit support tickets, and chat with support via Happychat. These features were however seldom used, and this section became a maintenance burden.  

### How to test the changes in this Pull Request:

1. Load the Dashboard, observe the Support wizard is no more
2. Define `NEWSPACK_SUPPORT_EMAIL` as an environment variable, observe a "mailto" link using it in Newspack admin panel footer

### Other information:
